### PR TITLE
Accessibility bug fixes

### PR DESCRIPTION
**Pull request by gingi**
Various accessibility bugs, most related to VoiceOver, as well as improvements to localization.

### CODE REVIEW

File: src/@batch-flask/ui/form/form-field/form-field.html [hunk 1]
```diff
@@ -19,9 +19,9 @@
         <ng-content select="[blFormFieldSuffix]"></ng-content>
     </div>
 </div>
-<div class="hints-container" *ngIf="hints.length > 0">
+<div class="hints-container" aria-live="polite">
```
Comment (dpwatrous) [inline src/@batch-flask/ui/form/form-field/form-field.html:22]: Does this result in a visual change now that hints-container is always visible?

File: src/app/components/account/details/programatic-usage/aad-credentials-picker/generate-aad-app-secret/generate-aad-app-secret.html [hunk 2]
```diff
@@ -31,11 +31,11 @@ <h4>Secret</h4>
         <bl-aad-app-secret-picker [formControl]="secret">
         </bl-aad-app-secret-picker>
 
-        <bl-button type="wide" color="primary" [action]="addNewSecret" matTooltip="Add a new secret without modifing the existing ones">
+        <bl-button type="wide" color="primary" [action]="addNewSecret" title="AB#558 Add a new secret without modifing the existing ones">
```
Comment (dpwatrous) [inline src/app/components/account/details/programatic-usage/aad-credentials-picker/generate-aad-app-secret/generate-aad-app-secret.html:34]: Looks like a copy/paste error or something with the title

File: src/app/components/pool/details/pool-cost-card/pool-cost-card.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
             [attr.aria-label]="label">
         </canvas>
     </div>
-    <div class="total">
+    <div class="total" aria-live="polite">
```
Comment (dpwatrous) [inline src/app/components/pool/details/pool-cost-card/pool-cost-card.html:10]: Can we just remove this warning instead? April 2019 is a long time ago

**Review by dpwatrous — CHANGES_REQUESTED** [review]

File: src/@batch-flask/ui/form/form-field/form-field.html (hunk 1 above)
Comment (gingi) [reply src/@batch-flask/ui/form/form-field/form-field.html:22]: No, I checked. The `aria-live` specification is that the content _within_ such regions is announced when it changes, not when those regions are added to the DOM. So they need to exist in place, even as empty containers, for AT to properly announce content changes.

File: src/app/components/account/details/programatic-usage/aad-credentials-picker/generate-aad-app-secret/generate-aad-app-secret.html (hunk 2 above)
Comment (gingi) [reply src/app/components/account/details/programatic-usage/aad-credentials-picker/generate-aad-app-secret/generate-aad-app-secret.html:34]: Oops, that was a little temporary note to myself.

File: src/app/components/pool/details/pool-cost-card/pool-cost-card.html (hunk 3 above)
Comment (gingi) [reply src/app/components/pool/details/pool-cost-card/pool-cost-card.html:10]: Good call. Removing section.

**Review by dpwatrous — APPROVED** [review]

**Comment by cRui861** [comment]
Fixes #2634

**Pull request merged**